tests/aws/services/cloudformation/resources/test_s3.py

```diff
@@ -28,10 +31,11 @@ def test_bucketpolicy(deploy_cfn_template, aws_client):
         ),
         template_mapping={"include_policy": False},
     )
-    with pytest.raises(Exception) as err:
-        aws_client.s3.get_bucket_policy(Bucket=bucket_name).get("Policy")
+    with pytest.raises(ClientError) as err:
+        aws_client.s3.get_bucket_policy(Bucket=bucket_name)
+    snapshot.match("no-policy", err.value.response)
 
-    assert err.value.response["Error"]["Code"] == "NoSuchBucketPolicy"
+    # assert err.value.response["Error"]["Code"] == "NoSuchBucketPolicy"
```

Comment:
Should we remove that line entirely instead of commenting it out as it is part of the snapshot.

Reply:
Yes, definitely! good catch, was a bit too fast here 😅